nextcord/message.py

```diff
@@ -516,8 +517,8 @@ class Message(Hashable):
     author: Union[:class:`Member`, :class:`abc.User`]
         A :class:`Member` that sent the message. If :attr:`channel` is a
         private channel or the user has the left the guild, then it is a :class:`User` instead.
-    content: :class:`str`
-        The actual contents of the message.
+    content: Optional[:class:`str`]
```

Comment:
This shouldn't be Optional.

If message_content is disabled, Discord will return an empty string when message content is not available.

Reply:
actually resolved.